apps/samenvvv/pages/announcements.tsx

```diff
@@ -0,0 +1,191 @@
+import { GetServerSideProps } from 'next'
+import Head from 'next/head'
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { MDXRemoteSerializeResult } from 'next-mdx-remote'
+import { serialize } from 'next-mdx-remote/serialize'
+import { NextSeoProps } from 'next-seo'
+import { useTranslation } from 'react-i18next'
+
+import i18nConfig from '..//next-i18next.config'
+import { Layout } from '../components'
+import { HashtagAnnouncement } from '../components/HashtagAnnouncement' //TODO fix import style
+
+type HashtagEventsProps = {
+  seo: NextSeoProps
+  source: MDXRemoteSerializeResult
+  hashtag: Hashtag
+  hasStarted: boolean
+  link: string
+}
+
+const AnnouncementEvent: FC<HashtagEventsProps> = ({
+  seo,
+  source,
+  hashtag,
+  hasStarted,
+  link,
+}) => {
+  const { t } = useTranslation()
+
+  return (
+    <Layout seo={seo} isDark>
+      <Head>
+        <meta
+          property="twitter:image:src"
+          content={seo.openGraph.images[0].url}
+        />
+      </Head>
+      <Hero title={seo.title as string} isFullHeight={false} />
+      <Container overflowX="hidden">
+        {/* {source && (
+          <Box my={8} maxW="container.md" mx="auto" textAlign="center">
+            <Markdown source={source} />
+          </Box>
+        )} */}
+        {!hasStarted ? (
+          <HashtagAnnouncement hashtag={hashtag} link={link} />
+        ) : (
+          <>
+            <Text color={'primary'} m={4}>
+              {t('join-previous-hashtag')}
+            </Text>
+            <Link href={link}>
+              <Text fontWeight={'bold'} color={'primary'} m={4}>
+                {t('join-link')}
+              </Text>
+            </Link>
+          </>
+        )}
+      </Container>
+    </Layout>
+  )
+}
+
+export default AnnouncementEvent
+
+export const getServerSideProps: GetServerSideProps = async context => {
+  const locale = context.locale as StrapiLocale
+  const queryClient = new QueryClient()
+
+  const ssrTranslations = await serverSideTranslations(
+    locale as StrapiLocale,
+    ['common'],
+    i18nConfig,
+  )
+
+  const args: SearchModelArgs<Hashtag> = {
+    url: 'api/hashtags',
+    locale,
+    statuses: ['approved'],
+    pageSize: 1,
+  }
+
+  const queryKey = Object.values(args)
+
+  await queryClient.prefetchQuery(queryKey, () => searchModel<Hashtag>(args))
+
+  const hashtagsResponse =
+    queryClient.getQueryData<StrapiCollectionResponse<Hashtag[]>>(queryKey)
+
+  const hashtag = hashtagsResponse?.data[0]
+
+  const title = {
+    en: 'Hashtag Announcement',
+    nl: 'Hashtag Announcement',
+    tr: 'Hashtag Duyurulari',
+  }
+
+  const description = {
+    en: '',
+    nl: '',
+    tr: '',
+  }
+
+  const twitterHandle = {
+    en: '@samenvvvEn',
+    nl: '@samenvvv',
+    tr: '@samenvvvTr',
+  }
+
+  if (!hashtag) {
+    return {
+      props: {
+        ...ssrTranslations,
+        seo: { title: title[locale] },
+        hashtag: null,
+      },
```

Comment:
It's my mistake. We shouldn't render the page if hashtag is not found. Otherwise we might show some announcement content without any actual hashtag. This case doesn't make any sense. 

```suggestion
      notFound: true,
```